kura/org.eclipse.kura.wire.component.provider/src/main/java/org/eclipse/kura/internal/wire/asset/WireAsset.java

```diff
@@ -209,8 +210,8 @@ protected String getFactoryPid() {
      * <li>Perform all write operations on associated writing channels</li>
      * <ul>
      *
-     * Both of the aforementioned operations are performed as soon as it timer
-     * wire component is also triggered.
+     * Both the aforementioned operations are performed as soon as this Wire Component 
+     * receives emit events triggered by {@link Timer} Wire Component.
```

Comment:
I don't think that the only source of events for an asset is the timer. In my opinion, we should stay general.